.circleci/config.yml

```diff
@@ -85,6 +85,47 @@ commands:
           command: |
             cd ./LBHFSSPublicAPI/
             sls deploy --stage <<parameters.stage>> --conceal
+  migrate-database:
+    description: "Migrate database"
+    parameters:
+      stage:
+        type: string
+    steps:
+      - *attach_workspace
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install Unzip
+          command: apt-get update && apt-get install unzip
+      - run:
+          name: Install AWS CLI
+          command: |
+            curl -L -o awscliv2.zip "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip"
+            unzip awscliv2.zip
+            ./aws/install
+      - run:
+          name: Install Session Manager plugin
+          command: |
+            curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_64bit/session-manager-plugin.deb" -o "session-manager-plugin.deb"
+            dpkg -i session-manager-plugin.deb
+      - run:
+          name: Install dotnet ef core
+          command: dotnet tool install dotnet-ef --tool-path ./dotnet-ef-local/
+      - run:
+          name: SSH into RDS and migrate database
+          command: |
+            aws ssm get-parameter --name "/platform-apis-jump-box-pem-key" --output text --query Parameter.Value > ./private-key.pem
```

Comment:
I'm not 100% sure but i THINK If you were to log in to AWS prior to this step, and the credentials/role you use has the relevant permissions to use session manager to connect to the EC2 you **might** not need to worry about storing a pem key anywhere.

Reply:
I will check this out and see.  Full disclosure, credit to the resident-contact-api project for this.